.github/workflows/ios-ci.yml

```
@@ -0,0 +1,71 @@
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: TestResults
+          path: TestResults.xcresult
+
+      - name: Summarize test run
+        if: always()
+        run: |
+          if [ -d "TestResults.xcresult" ]; then
+            SUMMARY=$(xcrun xcresulttool get --path TestResults.xcresult --format json | python3 - <<'PY'
+import json
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 52: could not find expected ':'

(syntax-check)